utils/translate_wav.sh

```diff
@@ -0,0 +1,210 @@
+#!/bin/bash
+
+# Copyright 2019 Nagoya University (Takenori Yoshimura)
+#           2019 RevComm Inc. (Takekatsu Hiramura)
```

Comment:
Change here?

Reply:
IMO these copyrights are not suitable nowadays and difficult to update. If you (@kan-bayashi, @hiratake55) do not mind, I suggest to use `ESPnet authors` instead like other OSS: https://github.com/tensorflow/tensor2tensor/blob/master/tensor2tensor/problems.py#L2

Reply:
@ShigekiKarita I do not mind this change. Please change the lines to more general way like `ESPnet authors`.